package.json

```diff
@@ -52,6 +52,7 @@
     "clsx": "^2.1.1",
     "crypto-js": "^4.2.0",
     "dayjs": "^1.11.11",
+    "ethers": "^6.13.2",
```

Comment:
Would it be possible to use just viem instead of adding ethers?
As we already have Viem  and it is pretty much the same library

Reply:
This is not a reason and moreover ethers.js more popular lib and more robust on this time.